include/etcpal/log.h

```diff
@@ -41,11 +41,18 @@
  * etcpal_create_syslog_str() and etcpal_create_human_log_str() to create log messages with a header
  * defined by the Syslog protocol or with a human-readable header defined by ETC.
  *
+ * The human-readable format consists of an ISO timestamp with the separating `T` replaced by a
+ * space, followed by a string indicating the priority and the message:
+ * ```
+ * 1970-01-01 00:00:00.001-06:00 [CRIT] This is a log message!
```

Comment:
We may want to run this format by duffy, since he had opinions about logging the last time.  I'm mostly concerned about logchopper -- I don't know what level it parses at.